packages/core/src/BreadCrumb/BreadCrumb.js

```diff
@@ -17,8 +16,13 @@ const Page = ({ Component, onClick, elem, classes }) => (
     data={elem}
     classes={{ a: classes.a }}
   >
-    <HvTypography noWrap component="div" variant="highlightText" className={classes.link}>
-      <HvOverflowTooltip data={startCase(elem.label)} />
+    <HvTypography
+      noWrap
+      component="div"
+      variant="highlightText"
+      className={clsx(classes.link, classes.label)}
```

Comment:
I would reuse the existing classes (not growing the API), but this is fine too 👍